MANUAL.md

```diff
@@ -66,16 +66,16 @@ a variable. Instead, all tokens represented by a `$` followed by letters are
 considered to be disk volumes in the GUARDIAN space. The `$V` form should not
 be used. The `$(VARIABLE)` form should be used for all variable references.
 
-### Special Variables
-
-GUARDIAN MAP DEFINE values, if they are in the GMake process context, can be
-referenced using the form:
-
-    $=DEFINE
-
-In this case, the variable will be substituted with the value of the `FILE`
-attribute of the MAP DEFINE.
-
+<!-- ### Special Variables -->
+<!--  -->
+<!-- GUARDIAN MAP DEFINE values, if they are in the GMake process context, can be -->
+<!-- referenced using the form: --> 
+<!--  -->
+<!--     $=DEFINE -->
+<!--  -->
+<!-- In this case, the variable will be substituted with the value of the `FILE` -->
+<!-- attribute of the MAP DEFINE. -->
+<!--  -->
```

Comment:
I'd remove it rather than uncommenting it. After all .md files are (supposed to be) human readable

Reply:
> 
> 
> I'd remove it rather than uncommenting it. After all .md files are (supposed to be) human readable

I'm going to be adding it back in soon, but just have not decided how to apply the change from the GMAKEN fork. There were a lot of structural changes between 4.1 and 4.3, but this isn't one of them. I might get to it sometime around the winter break.